specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -30,7 +30,7 @@ _Jakarta™ Messaging Specification, Version 3.0_. Available at: _https://jakart
 
 _Jakarta™ Transaction Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/transactions/2.0_
 
-_Java™ Transaction Service, Version 1.0_ (JTS specification)_. Available at: _https://www.oracle.com/technetwork/java/javaee/jts-spec095-1508547.pdf_
+_Java™ Transaction Service (JTS) Specification, Version 1.0 (Draft)_. Available at: _http://download.oracle.com/otn-pub/jcp/7309-jts-1.0-spec-oth-JSpec/jts1_0-spec.pdf_
```

Comment:
I think we should change the "http://" to "https://" I am going to raise a PR for the same on Jakarta Transactions

Reply:
Doh!  I saw that when I was making my other edits, but I must have missed pushing the change.  Sorry and thanks!